### PR TITLE
Switch to SweetAlert2

### LINKED ISSUE #9413: Consider to switch to SweetAlert2
SweetAlert is not so active.

### PR DESCRIPTION
**Pull request by realLiangshiwei**
Resolve https://github.com/abpframework/abp/issues/9413

### CODE REVIEW

File: framework/src/Volo.Abp.AspNetCore.Mvc.UI.Packages/Volo/Abp/AspNetCore/Mvc/UI/Packages/SweetAlert2/Sweetalert2ScriptContributor.cs [hunk 1]
```diff
@@ -3,14 +3,14 @@
 using Volo.Abp.AspNetCore.Mvc.UI.Packages.Core;
 using Volo.Abp.Modularity;
 
-namespace Volo.Abp.AspNetCore.Mvc.UI.Packages.SweetAlert
+namespace Volo.Abp.AspNetCore.Mvc.UI.Packages.SweetAlert2
 {
     [DependsOn(typeof(CoreScriptContributor))]
-    public class SweetalertScriptContributor : BundleContributor
```
Comment (berkansasmaz) [inline framework/src/Volo.Abp.AspNetCore.Mvc.UI.Packages/Volo/Abp/AspNetCore/Mvc/UI/Packages/SweetAlert2/Sweetalert2ScriptContributor.cs:9]: Maybe we should provide backward compatibility here what do you think?

Comment (realLiangshiwei) [reply framework/src/Volo.Abp.AspNetCore.Mvc.UI.Packages/Volo/Abp/AspNetCore/Mvc/UI/Packages/SweetAlert2/Sweetalert2ScriptContributor.cs:9]: SweetAlert2 is not the next version of SweetAlert,  it's a new project and is not compatible with SweetAlert. we cannot make them exist in the same project at the same time

**Comment by hikalkan** [comment]
@realLiangshiwei can you please resolve the conflicts so we merge this. This is a breaking change but won't effect most of the applications, they just run `abp install-libs` again after 5.0 upgrade.

**Comment by hikalkan** [comment]
@maliming please test and merge this after resolving conflicts.

**Comment by realLiangshiwei** [comment]
Conflict resolved

**Review by maliming — APPROVED** [review]

**Pull request merged**